linkerd/proxy/identity-client-metrics/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "linkerd-proxy-identity-client-metrics"
```

Comment:
Here's an alternative approach to this (against main) https://github.com/linkerd/linkerd2-proxy/pull/2617 -- updating these metrics to reflect best practices (e.g. _total vs _count); it also means neither clients need to be explicitly aware of these metrics.